code/modules/client/preferences/loadout/loadout_item/mask.dm

```diff
@@ -34,3 +34,14 @@
 /datum/loadout_item/mask/bandana_skull
 	path = /obj/item/clothing/mask/bandana/skull
 
+/datum/loadout_item/mask/breath_mask
+	path = /obj/item/clothing/mask/breath
+
+/datum/loadout_item/mask/breath_mask_vox
+	path = /obj/item/clothing/mask/breath/vox
+
+/datum/loadout_item/mask/security_hailer
+	path = /obj/item/clothing/mask/gas/sechailer
+	restricted_roles = list(JOB_SECURITY_OFFICER,JOB_HEAD_OF_SECURITY,JOB_WARDEN)
+	customization_flags = CUSTOMIZE_NAME_DESC //has no parts to recolor
```

Comment:
Sec gets these in their vendor, why do they need a loadout item for it?

Reply:
i forgot it didn't have recoloring while putting it in load out, i think i confused it with the swat version

Reply:
Any item can be color rotated, for the record. Up to you if you want it.

Reply:
i mean i tested the security hailer with color rotation it didn't do anything so no real point to have it in loadout

Reply:
Oh its probably because its black.